nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -220,7 +220,7 @@ let
         gzip_types ${lib.concatStringsSep " " compressMimeTypes};
       ''}
 
-      ${optionalString cfg.recommendedZstdSettings ''
+      ${optionalString cfg.experimentalZstdSettings ''
```

Comment:
What other issues does this module has beyond the ordering issue that is solved now?

Reply:
Unfortunately there are still issues remaining that haven't been addressed for a while, like https://github.com/tokers/zstd-nginx-module/pull/23. The last release is from 2023 even though the default branch contains an important bug fix. If we aren't properly maintaining the zstd module ourselves either, we also shouldn't recommend it. I've seen on multiple occasions that people mindlessly turn on all our recommended nginx options. There are still [ordering issues if brotli and zstd are enabled](https://github.com/tokers/zstd-nginx-module/pull/44) which are both recommend by our nginx module.